src/Disks/IDiskRemote.h

```diff
@@ -161,21 +180,12 @@ struct IDiskRemote::Metadata
 
     using PathAndSize = std::pair<String, size_t>;
 
-    /// Remote FS (S3, HDFS) root path.
-    const String & remote_fs_root_path;
-
     /// Disk path.
     const String & disk_path;
 
-    /// Relative path to metadata file on local FS.
-    String metadata_file_path;
-
     /// Total size of all remote FS (S3, HDFS) objects.
     size_t total_size = 0;
 
-    /// Remote FS (S3, HDFS) objects paths and their sizes.
-    std::vector<PathAndSize> remote_fs_objects;
```

Comment:
Now PathAndSize is not used in this struct, let's remove ` using PathAndSize = ...` from line 181